pkg/http/http.go

```diff
@@ -67,6 +67,9 @@ type Request interface {
 	//
 	// For run plugin, the context controls cancellation.
 	Context() context.Context
+	// RespHeader returns the response header
```

Comment:
RespHeader returns an `http.Header` which allows you to add or set response headers before reaching the upstream.
Some built-in headers would not take effect, like ...